IBM/carbon-addons-iot-react
# chore(storybook): update to 5.3.17, remove uneeded workaround

**Summary**

- When we updated to `5.3.13` in https://github.com/IBM/carbon-addons-iot-react/pull/961 we had to add a workaround in the jest config to avoid an error. The fix was included in `5.3.17`, so we can update and remove it 🎉 

**Acceptance Test (how to verify the PR)**

- View the storybook deploy preview 
- Ensure stories and storybook features/addons/knobs are working as expected
